### PR TITLE
added lit : 0.11.0

### PR DESCRIPTION
**Pull request by Alston-Dias**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by seth-priya** [comment]
@prashantkhoje  please review

File: l/lit/Lit_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,81 @@
+#!/bin/bash -e
+#----------------------------------------------------------------------------
+#
+# Package       : llvm-project
+# Version       : llvmorg-11.0.1
+# Source repo   : https://github.com/llvm/llvm-project/
+# Tested on     : UBI 8.3
```
Comment (prashantkhoje) [inline l/lit/Lit_ubi_8.5.sh:7]: Script name suggests you used 8.5. Please change accordingly.

File: l/lit/Lit_ubi_8.5.sh [hunk 2]
```diff
@@ -0,0 +1,81 @@
+#!/bin/bash -e
```
Comment (prashantkhoje) [inline l/lit/Lit_ubi_8.5.sh:1]: Why is the folder and file named lit*?
Please avoid using capital letters in filename.

File: l/lit/Lit_ubi_8.5.sh [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+#----------------------------------------------------------------------------
+#
+# Package       : llvm-project
+# Version       : llvmorg-11.0.1
+# Source repo   : https://github.com/llvm/llvm-project/
+# Tested on     : UBI 8.3
+# Language      : Python
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : BulkPackageSearch Automation {maintainer}
```
Comment (prashantkhoje) [inline l/lit/Lit_ubi_8.5.sh:11]: Please add your name as maintainer.

File: l/lit/Lit_ubi_8.5.sh [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+                echo "$PACKAGE_URL $PACKAGE_NAME" > /home/tester/output/clone_fails
+                echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | $SOURCE | Fail |  Clone_Fails" > /home/tester/output/version_tracker
+        exit 1
+fi
+
+cd /home/tester/$PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+if ! python3 setup.py install; then
+    build_test_with_python2
+        exit 0
```
Comment (prashantkhoje) [inline l/lit/Lit_ubi_8.5.sh:66]: Please use correct indentation.

File: l/lit/Lit_ubi_8.5.sh (hunk 2 above)
Comment (prashantkhoje) [reply l/lit/Lit_ubi_8.5.sh]: Named as lit because, script focuses only on lit from llvm-project.

**Review by prashantkhoje — APPROVED** [review]
LGTM

File: l/lit/lit_ubi_8.5.sh [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+#----------------------------------------------------------------------------
+#
+# Package       : llvm-project
+# Version       : llvmorg-11.0.1
+# Source repo   : https://github.com/llvm/llvm-project/
+# Tested on     : UBI 8.5
+# Language      : Python
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Alston Dias <Vedang.Wartikar@ibm.com>
```
Comment (seth-priya) [inline l/lit/lit_ubi_8.5.sh:11]: Please correct the email id

File: l/lit/lit_ubi_8.5.sh [hunk 6]
```diff
@@ -0,0 +1,81 @@
+#!/bin/bash -e
+#----------------------------------------------------------------------------
+#
+# Package       : llvm-project
```
Comment (seth-priya) [inline l/lit/lit_ubi_8.5.sh:4]: this is misleading if we are not building the entire llvm-project tree, let's be specific

File: l/lit/lit_ubi_8.5.sh [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=llvm-project
```
Comment (seth-priya) [inline l/lit/lit_ubi_8.5.sh:21]: let's be specific with the package name if we are not building the entire tree

**Review by seth-priya — CHANGES_REQUESTED** [review]

**Review by seth-priya — APPROVED** [review]

**Pull request merged**